rpm/sailfish-setup.spec

```diff
@@ -25,5 +25,8 @@ if ! getent passwd sailfish-mdm >/dev/null ; then
 fi
 groupadd -rf sailfish-radio || :
 
+groupadd -f sailfish-mms || :
+usermod -a -G sailfish-mms nemo || :
```

Comment:
Hmm... If `sailfish-mms` group is supposed to control access to privileged functions of mms-engine then a non-privileged user shouldn't be part of this group. If `sailfish-mms` is not supposed to control access to privileged functions of mms-engine then why is it needed in the first place. I start having a feeling that I'm missing something.

Reply:
My understanding is that with this new group we can control what users can use MMS. Not used at the moment  but in the future with multiuser support there can be those too.

Reply:
If it's not used at the moment and there's no clear plan how it's going to be used, then let's forget it for now and get back to it when it starts making sense!

Reply:
Removing things is so much harder that adding them. Once you add something, it's a matter of time when something starts to depend on it and that's how unnecessary or simply wrong things pile up and litter the platform.